scripts/data/fullhog/gamecredits.txt

```diff
@@ -545,4 +545,38 @@ Matt "JJJ_Shabadoo" Matthews
 The #loki gang at irc.openprojects.net
 Bofa and Wendy from down at The Center
 
+
+*320 100 320 100 8
+!OPEN SOURCE CONTRIBUTORS
+Andrew "ei-grad" Grigorev
+Azamat H. "winterheart" Hackimov
+Bryan Perris
+C.W. "MaddTheSane" Betts
+Chris "tophyr" Sarbora
+Dan "danra" Raviv
+Daniel Gibson
+Edu "Arcnor" Garcia
+Eric Slutz
+GravisZro
+InsanityBringer
+Jacob "jcoby"
+Jan "jengelh" Engelhardt
```

Comment:
This is just my Unix/GitHub username, and not any callsign/nickname (Internet-centric or otherwise), please strip it.

Reply:
done